optuna/samplers/_nsga2/sampler.py

```diff
@@ -94,6 +96,7 @@ def __init__(
         *,
         population_size: int = 50,
         mutation_prob: Optional[float] = None,
+        crossover_name: str = "uniform",
```

Comment:
Could you add a document of this argument like others? I think it's better for users to understand the crossover variants.

Reply:
"name" here sounds a bit superfluous. I personally prefer just "crossover" since it's clear especially given the type.

Reply:
Looks good to start with just introducing a single new parameter 👍 .

Reply:
We're usually quite thoughtful when introducing new algorithms. Concretely, we often choose to raise experimental warnings before we're quite confident about either the API or the optimization performance. It communicates to the user that it should be used with some degree of caution. Thus, I'm wondering if we should do the same thing here, as for the first introduction. For reference, it'd be implemented similar to https://github.com/optuna/optuna/blob/master/optuna/samplers/_tpe/sampler.py#L262-L267. What do you think?